packages/frontend/components/auth/verification/phone-verification.tsx

```diff
@@ -1,22 +1,24 @@
 import { Button } from '@primitives';
 import { useTranslation } from 'next-i18next';
-import { useAppDispatch, useAuth, useTypedSelector } from '@hooks';
+import {
+  useAppDispatch,
+  // useAuth,
```

Comment:
remove comments

Reply:
also must be used, that's why i commented it